src/cli/playpass-client.ts

```diff
@@ -109,6 +109,20 @@ export default class PlaypassClient {
             });
     }
 
+    public async deleteGame(gameId: string): Promise<Game> {
+        return axios.request({
+            method: "DELETE",
+            url: `${this.host}/api/v1/games/${gameId}`,
+            headers: {
+                "X-API-TOKEN": this.authToken
+            }
+        })
+            .then((res: AxiosResponse<Game>) => {
+                // PlaypassClient.validateResponse(res, "Failed to delete game.");
```

Comment:
Do we not need to validate the response here?

Reply:
Fixed